gridt/src/app/home/expandable/expandable.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit, Input } from "@angular/core";
+
+@Component({
+  selector: "signal-message",
+  templateUrl: "./expandable.component.html",
+  styleUrls: ["./expandable.component.scss"],
+})
+export class SignalMessageComponent implements OnInit{
+
+  @Input() signalText;
```

Comment:
Within the context of our application I think a more suitable name is message text. Very banal change, but I think it helps comprehension.

Reply:
Why is that? Every time someone talked about giving a message to the followers it was called signaling. So why is signal message worse then message text?